backend-opal/src/server/init.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+require('dotenv').config();
```

Comment:
@hondahmd this should be at the top of the `main.js` if you want to separate it

Reply:
@adamgyulavari So I should put the init of `dotenv` at the beginning of `main.js` instead?